packages/addon-dev/src/rollup-declarations.ts

```diff
@@ -1,17 +1,21 @@
+import type { Plugin } from 'rollup';
 import execa from 'execa';
 import walkSync from 'walk-sync';
 import { readFile, writeFile } from 'fs/promises';
 
-export default function rollupDeclarationsPlugin(declarationsDir: string) {
+export default function rollupDeclarationsPlugin(
+  declarationsDir: string
+): Plugin {
   let glintPromise: Promise<void>;
 
   return {
     name: 'glint-dts',
-    buildStart: () => {
+    buildStart() {
```

Comment:
had to change this to method syntax to get access to `this`, unsure of the impact that might have. Other plugins seem to do this as well